ground/src/main/java/com/google/android/ground/ui/startup/StartupFragment.kt

```diff
@@ -51,11 +50,14 @@ class StartupFragment : AbstractFragment() {
     super.onResume()
     showProgressDialog(R.string.initializing)
     viewLifecycleOwner.lifecycleScope.launch {
-      try {
-        viewModel.initializeLogin()
-      } catch (t: Throwable) {
-        onInitFailed(t)
-      }
+      val success =
+        try {
+          viewModel.initializeLogin()
+        } catch (t: Throwable) {
+          Timber.e(t, "Failed to initialize login")
```

Comment:
initializeLogin already logs errors, so we shouldn't also need to do it here

Reply:
done